packages/server/lib/middleware/security.ts

```diff
@@ -0,0 +1,51 @@
+import { basePublicUrl, baseUrl } from '@nangohq/utils';
+import type { Router } from 'express';
+import helmet from 'helmet';
+
+export function securityMiddleware(app: Router): void {
```

Comment:
can we have this function return an array of middlewares than can then be used in the router. ex:
```
app.use(securityMiddlewares)
```

Reply:
indeed